tests/translations-stub.json

```diff
@@ -11,12 +11,18 @@
         "is": {
             "a": {
                 "number": 1,
-                "object": {"a": 2},
+                "object": {
+                  "a": 2
+                },
                 "boolean": false
             }
         },
         "interpolated": {
             "phrase": "Please replace %{item} with %{item} and %{another} thing"
+        },
+        "templated": {
+          "custom": "Please click <t name='custom'>here</t> to continue",
+          "self_enclosing": "Please click <t name='custom'/> to continue"
```

Comment:
is it possible to have an array here? 
could you please also test cases when we have mistakes in copies? for instance: `"Hello ${{username}"`, or `"Please click on <t name='custom>here<t>"`, etc...